### ISSUE TITLE
CLI is not loading formatting rules in 1.13.1+

### ISSUE THREAD

**Issue by travisMiehm** [detekt/detekt#3288]
## Expected Behavior
The formatting rules should be loaded and run when using versions 1.13.1+ of the cli and formatting jar. Using the sample `Foo.kt` below, I would expect the unused import to be reported an removed by autocorrect.

## Observed Behavior
The formatting rules are not loaded and run when using versions 1.13.1+ of the cli and formatting jar. `Foo.kt` is unmodified, and the unused import is not reported.

## Steps to Reproduce
Download the detekt cli and formatting plugin at version 1.13.1. Create a test file `Foo.kt:`
```
import kotlin.collections.ByteIterator

internal class Foo {
  fun bar = println("bar")
}
```

`config.yml`:
```
console-reports:
  active: true

formatting:
  active: true
  autoCorrect: true
  NoUnusedImports:
    active: true
    autoCorrect: true

```
`java -jar detekt-1.13.1.jar --config config.yml -i "Foo.kt" -p "detekt-formatting-1.13.1.jar" -ac --fail-fast --debug -dd;`

## Context
I run detekt as a git hook, and tried upgrading recently. I noticed that unused imports were no longer being removed automatically. `NoUnusedImports` is just the rule I was using to test with, but none of the formatting rules are running.

The debug output yields
>Loaded extensions:
>
>Registered rule sets:
>
>Loaded extensions:
>    io.gitlab.arturbosch.detekt.rules.documentation.LicenceHeaderLoaderExtension@2301b75
>    io.github.detekt.metrics.processors.ProjectSLOCProcessor@3104f7bd
>    io.github.detekt.metrics.processors.ProjectLOCProcessor@2f5c1332
>    io.github.detekt.metrics.processors.ProjectCLOCProcessor@7ec25216
>    io.github.detekt.metrics.processors.ProjectLLOCProcessor@7cab1508
>    io.github.detekt.metrics.processors.ProjectCognitiveComplexityProcessor@6a84bc3f
>    io.github.detekt.metrics.processors.ProjectComplexityProcessor@258ee7de
>    io.github.detekt.metrics.processors.PropertyCountProcessor@4930539b
>    io.github.detekt.metrics.processors.FunctionCountProcessor@6d171ce0
>    io.github.detekt.metrics.processors.ClassCountProcessor@22bd2039
>    io.github.detekt.metrics.processors.PackageCountProcessor@705f83a7
>    io.github.detekt.metrics.processors.KtFileCountProcessor@3bb8aabc
>    io.gitlab.arturbosch.detekt.cli.DetektProgressListener@6e1d4137
>Registered rule sets:
>
>.
>
>1 kotlin file was analyzed.

The empty "Registered rule sets" looks odd to me. When I run the same command but with 1.12.0, I see
>Registered rule sets: 
>    io.gitlab.arturbosch.detekt.formatting.FormattingProvider@5afa3c9

## Your Environment

* Version of detekt used: 1.13.1 (tried 1.14.2 as well, same result)
* Operating System and version: macOS Catalina 10.15.7

**Comment by arturbosch** [comment]
Hi,
I've used your test file and config.
Versions 1.14.2 and 1.15.0-RC2 both work for me as expected:

![Screenshot from 2020-12-14 21-01-02](https://user-images.githubusercontent.com/20924106/102129447-a4040f00-3e4f-11eb-9318-cd0893acbcb4.png)

**Comment by travisMiehm** [comment]
>Do you use the same formatting jar with both cli jars? It should work. Maybe one formatting.jar is corrupt?

No, I've been using the formatting.jar with the same version as the cli.jar that I'm using. So cli 1.12.0 with formatting.jar 1.12.0 and cli 1.13.1 with formatting.jar 1.13.1, and 1.14.2.

**Comment by travisMiehm** [comment]
I had been downloading the jar file from bintray, but just saw you've moved to maven recently. Downloading the 1.13.1 formatting.jar from maven cleared up the issue.

Thanks for the quick response, and sorry for the false alarm.
